cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -237,3 +237,83 @@ def test_service_cloud_tenant_quota_with_default_entry_point(request, appliance,
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert provision_request.row.reason.text == "Quota Exceeded"
+
+
+def create_instance(appliance, provider, template_name):
+    instance = appliance.collections.cloud_instances.instantiate(random_vm_name('pwr-c'),
+                                                                 provider,
+                                                                 template_name)
+    if not instance.exists_on_provider:
+        instance.create_on_provider(allow_skip="default", find_in_cfme=True)
+    return instance
+
+
+@pytest.fixture(scope="function")
+def testing_instance(appliance, provider, small_template, setup_provider):
```

Comment:
I think these two methods can be merged into one . I don't see any significance of deleting the instance in separate method.

     
    @pytest.fixture(scope="function")
    def create_instance(appliance, provider, small_template, setup_provider):
        """ Fixture to provision instance on the provider"""
        instance = appliance.collections.cloud_instances.instantiate(random_vm_name('pwr-c'),
                                                                 provider,
                                                                 small_template.name)
        if not instance.exists_on_provider:
            instance.create_on_provider(allow_skip="default", find_in_cfme=True)
        yield instance
        instance.cleanup_on_provider()